src/java.base/share/classes/java/security/Security.java

```diff
@@ -39,6 +45,10 @@
 
 import sun.security.jca.*;
 
+/*[IF CRIU_SUPPORT]*/
+import openj9.internal.criu.InternalCRIUSupport;
+/*[ENDIF] CRIU_SUPPORT*/
```

Comment:
I think this also needs
```
import openj9.internal.criu.security.CRIUConfigurator;
```

Reply:
I already cherry-picked [PR openj9-openjdk-jdk11#521](https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/521).

Did you mean [PR openj9-openjdk-jdk11#523](https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/523/files)?

Reply:
Sorry, yes, I meant jdk11#523.

Reply:
added [openj9-openjdk-jdk11#523](https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/523) and [openj9-openjdk-jdk11#524](https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/524)